CHANGELOG.md

```diff
@@ -11,8 +11,10 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 
 ### Added Features
 
-- ROX-25625: RHCOS Node Scanning with Scanner V4 can now detect vulnerabilities for the containerized image of the RHCOS itself.
-- ROX-26849: Introduce report caching for RHCOS Node Indexing.
+- ROX-26847: RHCOS Node Scanning with Scanner V4
+    - ROX-27719: is now enabled by default on all secured clusters and will be preferred over the Stackrox Scanner if Scanner V4 is installed and connected to Central.
+    - ROX-25625: can now detect vulnerabilities for the containerized image of the RHCOS itself.
```

Comment:
```suggestion
    - ROX-25625: can now detect vulnerabilities for its own containerized RHCOS image.
```
[nit] better wording

Reply:
It is not it's own image 😅 RHCOS runs on the nodes and Scanner V4 runs in an image. Both are RHEL9-based, but I don't think that is enough to say that Scanner V4's own image is RHCOS.

Reply:
But I get the confusion caused by the word `itself`. What I meant by this is that there is no rpm package called `rhcos`, so the previous scanner V4 (in 4.6) could not report vulns on it. Now we add that, so `rhcos` is represented by a package named the same as the operating system (this is intended, as VEX publishes data under this name). 
Having said that, would you like to propose another rephrasing maybe?

Reply:
@jschnath I would like to merge Today. Pinging you in case you want to propose another rephrasing. 
If not, I would merge it as it is - it does not need to be ideal, as release notes are not copy-pasted 1:1, but edited by the doc writers before publishing.